process-automation/latest/admin/release.md

```diff
@@ -199,3 +199,19 @@ To update the runtime version.
 > **Note:** You can also update the runtime version of an application when you upgrade. For more see [Upgrade](#upgrade)
 
 ![update runtime]({% link process-automation/images/update-runtime.png %})
+
+### Logs
+
+You can view Deployment Service Logs, Runtime Bundle Logs, and Process Storage Logs to help understand any errors you may be having with your installation. The Deployment Service Logs are always available because they belong to a shared service. The logs use different blue highlighting to indicate the app being viewed. The Runtime Bundle Logs and the Storage Logs are unavailable if the app is not deployed correctly.
+
+To view the logs.
+
+1. Log into the Alfresco Admin App.
+
+2. Expand **Devops** from the left pane and select **Application Instances**.
```

Comment:
```suggestion
2. Expand **DevOps** from the left pane and select **Application Instances**.
```

Reply:
its not written like that! cheers :-)

Reply:
Ok - I was following what's at the top of the page.